tests/UI/pages/commonPage.js

```diff
@@ -243,11 +243,12 @@ module.exports = class CommonPage {
   /**
    * Go to Page and wait for navigation
    * @param selector
+   * @param waitUntil
    * @return {Promise<void>}
    */
-  async clickAndWaitForNavigation(selector) {
+  async clickAndWaitForNavigation(selector, waitUntil = 'load') {
     await Promise.all([
-      this.page.waitForNavigation(),
+      this.page.waitForNavigation(waitUntil),
```

Comment:
What is `waitUntil` here ? I don't see it in the method signature

Reply:
It's the navigation event to wait, I will add it in docs.

Reply:
Oh yes I needed to open the full file to see it, It wan't visible on the review page :/